pages/linux/hcitool.md

```diff
@@ -0,0 +1,36 @@
+# hcitool
+
+> Monitor & Configure Bluetooth connections.
+> Send special commands to Bluetooth devices.
+
+- Scan for remote devices:
+
+`hcitool scan`
+
+- Output name of the device with its mac address:
+
+`hcitool name {{bdaddr}}`
+
+- Fetch info of a remote bluetooth device:
+
+`hcitool info {{bdaddr}}`
+
+- Check link quality:
+
+`hcitool lq {{bdaddr}}`
+
+- Display transmit power level:
+
+`hcitool tpl {{bdaddr}} {{type}}`
```

Comment:
Even after reading the man page, I don't really understand what `type` is.

Reply:
```
       tpl <bdaddr> [type]
              Display transmit power level for the connection to the device with Bluetooth address bdaddr.

              The type can be 0 for the current transmit power level (which is default) or 1 for the maximum transmit power level.
```
Yeah, it's a bit confusing even in man-page.
What I've understood from this explanation is, if we specify 0, then the
tpl will be set to the current transmit power level instead of increasing
or decreasing it's intensity of transmission, whereas maximum tpl will
be applied when this type is set to 1.

Even though this is not a good explanation
on <type>, I hope it makes better sense...

Reply:
If specifying `type` modifies it instead of displaying it we should remove it. Or change the description.

Reply:
How about changing the placeholder of type?
```console
hcitool tpl {{bdaddr}} {{0|1}}
```

Reply:
But this still modifies the transmit power level instead of printing it. Either we change the description to `Modify the transmit power level` or we remove this.

Reply:
Ahh, now I get it.
Let's change the description.

Reply:
Changes committed!